build.gradle

```diff
@@ -18,13 +18,13 @@ plugins {
     id 'com.jfrog.bintray' version '1.8.4'
 }
 
-version = '2.17.7'
+version = '2.18.0'
```

Comment:
Not sure if it doesn't need to be 3.0.0 as it's not backwards compatible with the previous version...